### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by Chifrica** [OpenSPP/documentation#9] [General]
Hello 👋 am thrilled to be part of this program. My name is Onwunali Blessing one of the Community Manager of OpenSPP, I welcome you all into this great house.

**Comment by ofonime1** [comment]
Hey Blessing @Chifrica and everyone.
My name is Ofon Brown.
It's good to be here 
Cheers.

**Reply by Chifrica** [reply]
Actually it's good to be here. Welcome @ofonime1

**Comment by SlimmyTee** [comment]
Hello Everyone, 
I am excited to be here, 
My name is Daniel Umoren

**Reply by Chifrica** [reply]
Nice to have @SlimmyTee

**Reply by ofonime1** [reply]
Welcome onboard

**Comment by FaithNjah** [comment]
Hi everyone, I am Faith Njah. I look forward to making meaningful contributions to this program.

**Reply by Chifrica** [reply]
You're welcome @FaithNjah

**Reply by ofonime1** [reply]
Looking forward 😁

**Comment by kiche-allan** [comment]
Hello Everyone. Good to see youll here. 
Am Allan Kiche from Kenya

**Reply by Chifrica** [reply]
Good to see here also @kiche-allan

**Reply by ofonime1** [reply]
I agree totally.

**Comment by Kasope** [comment]
Hello Everyone, Glad to be here,
I'm Fatima Olasunkanmi-Ojo, looking forward to contributing to OpenSPP project

**Reply by Chifrica** [reply]
Welcome @Kasope nice to meet you

**Comment by Pheogrammer** [comment]
Hello everyone,

am more than happy to be here. My name is Alfeo Raymond from Tanzania

**Reply by ofonime1** [reply]
Excited to meet you.

**Reply by Chifrica** [reply]
Nice to have you 🎉

**Comment by Chifrica** [comment]
Hello everyone, hope we all having a wonderful days.
I will like us to keep this place active by sharing ideas, bringing suggestion of how we can make this project of a great one.

**Reply by Chifrica** [reply]
Your contribution can be of help to others.

**Comment by aTfure** [comment]
Hello team, 
My name is Fredrick Onyango. I am more than happy to be here. Hope we learn from each others.

**Reply by Chifrica** [reply]
Welcome @aTfure nice to have you here. Looking forward to learn from you, and also offer what i have.

**Comment by Luckyjoseph** [comment]
Hello everyone,
My name is Lucky Joseph Hassan and i am excited to be here.
I cant wait to see the amazing things we could do together as a team.

**Reply by Chifrica** [reply]
You're welcome @Luckyjoseph  nice to have you here and hoping to achieve something great with you.
